skbio/util/_testing.py

```diff
@@ -6,14 +6,13 @@
 # The full license is in the file COPYING.txt, distributed with this software.
 # ----------------------------------------------------------------------------
 
-import os
 import inspect
-import warnings
-
 import nose
 import numpy as np
 import numpy.testing as npt
+import os
 import pandas.util.testing as pdt
+import warnings
```

Comment:
why this change? Not to clutter the imports, we typically put the import of builtin module in top; an empty line; the import of external pkg; an empty line; and then the import of the module in current pkg.